src/Core/src/Sts/ValueObject/PolicyDescriptorType.php

```diff
@@ -1,25 +1,17 @@
 <?php
 
-namespace AsyncAws\Core\Sts\Input;
+namespace AsyncAws\Core\Sts\ValueObject;
 
 class PolicyDescriptorType
 {
-    /**
-     * The Amazon Resource Name (ARN) of the IAM managed policy to use as a session policy for the role. For more
-     * information about ARNs, see Amazon Resource Names (ARNs) and AWS Service Namespaces in the *AWS General Reference*.
```

Comment:
Doesn’t it make sense to have comments on properties now?

Reply:
Given the object is used in response, the docblock should at least be on getters.
Otherwise, user will loose IDE autocompletion.

![Screenshot from 2020-03-16 09-58-41](https://user-images.githubusercontent.com/578547/76739635-c3f58d80-676c-11ea-8c3a-c412ebe87704.png)

Or we could dupplicate dockbloc on each property/setters/getters

Reply:
Let's duplicate them for now. 
I believe they are needed for me to understand what I should do with the input.

Reply:
Changed my mind. When writing entities, I don't write docbloc on getters neither on setter. Using property is the right place...

I applyed the rule on Input, ValueObject and Result